app/src/main/res/layout/view_multi_action_add.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is this a `ConstraintLayout` masquerading as a `LinearLayout`? 😉

Reply:
Hehe could use a LinearLayout here for sure, but I actually prefer ConstraintLayouts :)

Reply:
I usually prefer to use `LinearLayout`s for these kinds of basic cases as they perform a little better and are less verbose (one `android:orientation` vs `n * 4` constraint declarations where `n` is the amount of views in the layout 🙂)

Reply:
Sure, but not as easy to change later on! Both works imo, just different styles :) Compose will remove all this nonsense anyway ;)

Reply:
Right you are 😉